docs/articles/documentation/guides/continuous-integration/circleci.md

```diff
@@ -6,139 +6,139 @@ redirect_from:
 
-![Clicking the Add Project Button](../../../images/circle-ci/add-new-project.png)
+For more information about reporters, see [Reporters](../concepts/reporters.md).
 
-You will see the list of GitHub projects associated with your account.
+Run the following command to install the packages locally in your project directory:
 
-Find `testcafe-ci-demo` and click **Setup project**.
-
-![Set Up Project](../../../images/circle-ci/set-up-project.png)
-
-The *Setup Project* screen will appear.
-
-In the **Operating System** section, select *Linux*. Then select *2.0* in the **Platform** section.
-
-![Select OS and Platform](../../../images/circle-ci/os-and-platform.png)
-
-In the **Language** section, select *Node*.
+```sh
+npm install --save-dev testcafe testcafe-reporter-xunit
+```
 
-![Select Language](../../../images/circle-ci/language.png)
+Commit the `package-lock.json` file generated by npm into the repository. This file allows CircleCI to use a fast `npm ci` command to install the dependencies.
 
-## Step 2 - Configuring the Build
+## Step 2 - Create a New CircleCI Project
 
-Create a `.circleci` directory in the `testcafe-ci-demo` repository fork and add a `config.yml` file there.
+Log in to CircleCI with your GitHub account.
 
-Use the content CircleCI provides by default with two changes:
+When you launch CircleCI for the first time, you start from an empty workspace. Click the **Add Project** button to create your first project.
 
-* Since you are going to perform browser testing, you need a virtual machine image that has all popular browsers pre-installed. So use the following image
+![Click the Add Project Button](../../../images/circle-ci/add-project.png)
 
-    ```yml
-    # Replace '10.14' with the latest Node.js LTS version
-    # available on Docker Hub
-    - image: circleci/node:10.14-browsers
-    ```
+The list of GitHub projects associated with your account appears.
 
-    Replace `10.14` with the latest available Node.js LTS version. See the [image's page](https://hub.docker.com/r/circleci/node/tags/) on Docker Hub for the list of available versions.
+Find `testcafe-ci-demo` and click **Set Up project**.
 
-* Add a step that imports test results to the end of the YAML. The results will be displayed in the **Test Summary** section.
+![Set Up Project](../../../images/circle-ci/set-up-project.png)
 
-    ```yml
-    - store_test_results:
-        path: /tmp/test-results
-    ```
+## Step 3 - Configure the Build
 
-The resulting YAML will look as follows.
+On the project configuration page, select **Node** as the target platform and paste the following configuration:
 
-```yml
-{% raw %}
-version: 2
+```yaml
+version: 2.1
+orbs:
+  node: circleci/node@4.1.0
 jobs:
-  build:
+  test:
     docker:
-      - image: circleci/node:10.14-browsers
-
-    working_directory: ~/repo
-
+      - image: circleci/node:latest-browsers
```

Comment:
as far as I figured, `circleci/node` is now obsolete and has been replaced with `cimg/node` (which I couldn't get to work with local browsers, however)

did you leave `circleci/node` on purpose?

Reply:
As [CircleCI docs state](https://circleci.com/developer/images/image/cimg/node#__next:~:text=You%20can%20use%20the%20orb%20to%20install%20a%20version%20of%20Google%20Chrome%20and%2For%20Firefox%20into%20your%20build.), the new `cimg/node` images come without browsers and require an orb to install browsers. I've rewritten the guide to use the new images.